prototypes/prototype-0/docs/verify-service-provider-api.swagger

```diff
@@ -0,0 +1,138 @@
+swagger: "2.0"
+info:
+  description: 'This a proposed API for the Verify Serivce Provider'
+  version: "1.0.0"
+  title: "Verify Service Provider"
+paths:
+  "/authn-request":
+    get:
```

Comment:
Our pattern has been to use GETs for AuthnRequests so we thought it should be consistent. However, on reflection, we agree that it should be a POST here as the endpoint isn't idempotent.